BossMod/Modules/RealmReborn/Trial/T09WhorleaterH/T09WhorleaterH.cs

```diff
@@ -0,0 +1,168 @@
+﻿using BossMod.Endwalker.Extreme.Ex7Zeromus;
+using BossMod.ReplayVisualization;
+using FFXIVClientStructs.FFXIV.Client.Game.UI;
```

Comment:
These all are hopefully not needed? Generally modules should not rely on game state, so that replays can be opened in uidev.